tgui/packages/tgui/interfaces/Who.jsx

```diff
@@ -11,20 +13,24 @@ import {
 import { Window } from '../layouts';
 
 export const Who = (props, context) => {
-  const { act, data } = useBackend(context);
+  const { act, data } = useBackend();
   const {
-    admin,
-    all_clients,
-    total_players = [],
-    additional_info = [],
-    factions = [],
-    xenomorphs = [],
+    base_data,
+    player_additional,
+    player_stealthed_additional,
+    factions_additional,
   } = data;
 
-  const [searchQuery, setSearchQuery] = useLocalState('searchQuery', '');
+  const total_players = mergeArrays(
+    base_data?.total_players,
+    player_additional?.total_players,
+    player_stealthed_additional?.total_players,
+  );
+
+  const [searchQuery, setSearchQuery] = useState('', '');
```

Comment:
```suggestion
  const [searchQuery, setSearchQuery] = useState('');
```
useState does not have an implementation taking two arguments.